frontend/src/components/search/SearchFilterAccordion/SearchFilterAccordion.tsx

```diff
@@ -86,7 +84,11 @@ const AccordionContent = ({
 
   return (
     <>
-      <ul className="usa-list usa-list--unstyled">
+      <ul
+        className={clsx("usa-list usa-list--unstyled", {
+          "maxh-mobile-lg": wrapForScroll,
```

Comment:
Not sure what this is trying to do. 

Is the intent to get rid of the horizontal scrollbar at the bottom of the filter? 

![image](https://github.com/user-attachments/assets/b2cb8a53-f266-4612-a9ed-71e629066ffe)

That might be achieved by changing `overflow-scroll` to `overflow-x-hidden` on the containing `div`, not the list itself. 

![image](https://github.com/user-attachments/assets/f1dffefa-2597-4cca-a1dc-d67365ef1155)

![image](https://github.com/user-attachments/assets/208c3098-b390-4fb9-9b1b-a7a90394ea55)

Reply:
But I'm wondering if the max height and scroll should go on the `.usa-accordion__content`, which has padding that's currently making the scrollable area smaller. The scroll bar could be flush with the accordion's border.